Sources/DeepLearning/Bindings/generate_wrappers.py

```diff
@@ -0,0 +1,717 @@
+# Copyright 2018 Google LLC
```

Comment:
Change this to the version that says `The TensorFlow Authors`?

Reply:
I updated it for all moved files.